calico-policies/private-network-isolation/calico-v3/us-south/allow-private-services-pods.yaml

```diff
@@ -74,6 +74,29 @@ spec:
       - 514
     protocol: UDP
     source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM ATracker API : Prod US South
+      - 166.9.58.136/32
+      - 166.9.48.211/32
+      - 166.9.51.140/32
+      # IBM ATracker API : Stage US South
```

Comment:
We don't put stage IPs in these policies.  We are in the process of creating a repo, https://github.ibm.com/alchemy-containers/armada-calico-policies that will hold the pre-production policies.

Reply:
Changes made in new commit.